wmn-data.json

```diff
@@ -3605,6 +3629,18 @@
         "m_code" : 404,
         "known" : ["malibu927", "L1zardQueen"],
         "cat" : "health",
+        "valid" : true
+       },        
+       {
+        "name" : "mymfans",
+        "uri_check" : "https://mym.fans/{account}",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : " • MYM</title>",
+        "m_string" : "Il n’y a rien ici…",

```

Comment:
This string is specific to you since you are probably viewing the site in a browser/commandline with French language/from French IP address space. In my browser, I see the error `There's nothing here...`. Perhaps use the title HTML tag `• MYM • Exclusive social network for creators &amp; fans</title>`

Reply:
@WebBreacher Thanks for your help, `Done!` re change of `m_string` on `mymfans`